include/pagmo/algorithms/mbh.hpp

```diff
@@ -366,7 +366,7 @@ class mbh : public algorithm
      * <tt>std::vector</tt> is a mbh::log_line_type containing: \p Fevals, \p Best, \p Violated, <tt>Viol. Norm</tt> and
      * \p Trial as described in mbh::set_verbosity().
      *
-     * @return an <tt> std::vector </tt> of mbh::log_line_type containing the logged values Fevals,
+     * @return an <tt> std::vector </tt> of mbh::log_line_type containing the logged values Fevals, Best,
```

Comment:
The extra spaces around ``std::vector`` should be removed, they look ugly in the sphinx output.